cmd/image-inspector.go

```diff
@@ -28,6 +29,8 @@ func main() {
 
 	flag.Parse()
 
+	inspectorOptions.AuthToken = os.Getenv("INSPECTOR_AUTH_TOKEN")
```

Comment:
if it is a shared secret, shouldn't it be a "Secret" that is shared between two pods? You can mount the Secret to multiple containers (which also have benefit of having one place to manage it)